### PR TITLE
[ALLUXIO-2813] Add CreateUfsFileOptions 

### PR DESCRIPTION
**Pull request by peisun1115**

### CODE REVIEW

File: core/server/worker/src/main/java/alluxio/worker/netty/DataServerWriteHandler.java [hunk 1]
```diff
@@ -150,6 +150,11 @@
      * Cancels the request.
      */
     abstract void cancel() throws IOException;
+
+    /**
+     * Cleans up the state.
```
Comment (calvinjia) [inline core/server/worker/src/main/java/alluxio/worker/netty/DataServerWriteHandler.java]: Could you add detail about when each of `close`, `cancel`, and `cleanup` should be called?

File: core/server/worker/src/main/java/alluxio/worker/netty/DataServerWriteHandler.java [hunk 2]
```diff
@@ -380,6 +384,17 @@ private void cancel() throws IOException {
     }
 
     /**
+     * Cancels this write.
```
Comment (calvinjia) [inline core/server/worker/src/main/java/alluxio/worker/netty/DataServerWriteHandler.java]: `Cleanup`?

File: core/server/worker/src/main/java/alluxio/worker/netty/DataServerWriteHandler.java (hunk 1 above)
Comment (peisun1115) [inline core/server/worker/src/main/java/alluxio/worker/netty/DataServerWriteHandler.java]: done

File: core/server/worker/src/main/java/alluxio/worker/netty/DataServerWriteHandler.java (hunk 2 above)
Comment (peisun1115) [inline core/server/worker/src/main/java/alluxio/worker/netty/DataServerWriteHandler.java]: done

**Pull request merged**